codegen/src/main/kotlin/com/android/designcompose/codegen/BuilderProcessor.kt

```diff
@@ -353,7 +340,7 @@ class BuilderProcessor(private val codeGenerator: CodeGenerator, val logger: KSP
             out.appendText("}\n\n")
 
             val objectName = className + "Doc"
-            out.appendText("object $objectName: $interfaceName {}\n\n")
+            out.appendText("val $objectName = $interfaceName()\n\n")
```

Comment:
I wonder if we still need this. Removing it would be a breaking change, but I can't think why statically instantiating an instance of it is a good thing.

I suppose the same question could have been asked about the original line too.

Reply:
It would mean all existing code would have to instantiate the object manually then wouldn't it? It'd break everything.